ivy/functional/backends/tensorflow/statistical.py

```diff
@@ -6,6 +6,18 @@
 # local
 import ivy
 
+def __new_var_fun(x, *, axis, correction, dtype):
```

Comment:
we instead use a single underscore for private functions, because the name mangling is quite annoying when debugging, and we don't really have a need for "protected" functions